### ISSUE TITLE
Allow separate persistence control for native window position and size

### ISSUE THREAD

**Issue by sourcebox** [emilk/egui#761]
I'll link to #591 here, where the topic was discussed already.

In conclusion, it should be possible to have the window position persistent, so the user can move the window whereever it should be and have it remembered on next launch. At the same time, the window size should be able to kept at a fixed size determined by the application at startup.

**Comment by quietvoid** [comment]
Agreed. I'm also interested in having control over the new `fullscreen` state.
